### ISSUE TITLE
How to get pretrained model and run these code?

### ISSUE THREAD

**Issue by Su210921** [thuanz123/realfill#25]
Hi,

Thank you very much for sharing your implementation!

I encountered an issue, and since I am new to the Diffuser library, I was hoping you could guide me on how to run these code and how to get the pretrained stable-diffusion-2-inpainting model. Or could you provide the pre-trained model?

I greatly look forward to your advice!!!

**Comment by thuanz123** [comment]
diffusers will download the pretrained model for you automatically. Usually some internet or connection issues prevent the automatic download. If you want to download by yourselves, you can use this link https://huggingface.co/stabilityai/stable-diffusion-2-inpainting

**Comment by Su210921** [comment]
Thank you very much!!!

**Comment by Su210921** [comment]
I'm getting the following error when I run it . Could you please teach me how to solve this problem?

usage: ipykernel_launcher.py [-h] --pretrained_model_name_or_path
                             PRETRAINED_MODEL_NAME_OR_PATH
                             [--revision REVISION]
                             [--tokenizer_name TOKENIZER_NAME]
                             --train_data_dir TRAIN_DATA_DIR
                             [--num_validation_images NUM_VALIDATION_IMAGES]
                             [--validation_steps VALIDATION_STEPS]
                             [--output_dir OUTPUT_DIR] [--seed SEED]
                             [--resolution RESOLUTION]
                             [--train_batch_size TRAIN_BATCH_SIZE]
                             [--num_train_epochs NUM_TRAIN_EPOCHS]
                             [--max_train_steps MAX_TRAIN_STEPS]
                             [--checkpointing_steps CHECKPOINTING_STEPS]
                             [--checkpoints_total_limit CHECKPOINTS_TOTAL_LIMIT]
                             [--resume_from_checkpoint RESUME_FROM_CHECKPOINT]
                             [--gradient_accumulation_steps GRADIENT_ACCUMULATION_STEPS]
                             [--gradient_checkpointing]
                             [--unet_learning_rate UNET_LEARNING_RATE]
                             [--text_encoder_learning_rate TEXT_ENCODER_LEARNING_RATE]
                             [--scale_lr] [--lr_scheduler LR_SCHEDULER]
                             [--lr_warmup_steps LR_WARMUP_STEPS]
                             [--lr_num_cycles LR_NUM_CYCLES]
                             [--lr_power LR_POWER] [--use_8bit_adam]
                             [--adam_beta1 ADAM_BETA1]
                             [--adam_beta2 ADAM_BETA2]
                             [--adam_weight_decay ADAM_WEIGHT_DECAY]
                             [--adam_epsilon ADAM_EPSILON]
                             [--max_grad_norm MAX_GRAD_NORM] [--push_to_hub]
                             [--hub_token HUB_TOKEN]
                             [--hub_model_id HUB_MODEL_ID]
                             [--logging_dir LOGGING_DIR] [--allow_tf32]
                             [--report_to REPORT_TO] [--wandb_key WANDB_KEY]
                             [--wandb_project_name WANDB_PROJECT_NAME]
                             [--mixed_precision {no,fp16,bf16}]
                             [--local_rank LOCAL_RANK]
                             [--enable_xformers_memory_efficient_attention]
                             [--set_grads_to_none] [--lora_rank LORA_RANK]
                             [--lora_alpha LORA_ALPHA]
                             [--lora_dropout LORA_DROPOUT]
                             [--lora_bias LORA_BIAS]
ipykernel_launcher.py: error: the following arguments are required: --pretrained_model_name_or_path, --train_data_dir
An exception has occurred, use %tb to see the full traceback.

SystemExit: 2
C:\anaconda\envs\RealFill\lib\site-packages\IPython\core\interactiveshell.py:3585: UserWarning: To exit: use 'exit', 'quit', or Ctrl-D.
  warn("To exit: use 'exit', 'quit', or Ctrl-D.", stacklevel=1)

**Comment by thuanz123** [comment]
which command you used for training ? seem like you used a custom code not my original command and code ?

**Comment by thuanz123** [comment]
However, this error is due to you haven't specify the pretained model path and also the directory for training. Maybe try sth like 

`--pretrained_model_name_or_path stabilityai/stable-diffusion-2-inpainting  --train_data_dir data/flowerwoman`

**Comment by Su210921** [comment]
Oh！ Thanks for your help!!
